scripts/cangen/cangen/traffic_calculation.py

```diff
@@ -0,0 +1,61 @@
+class BusLoadCalculator:
+ 
+    def __init__(self, can_speed = int):
+        """"
+        Initialize bus load calculator with the CAN Speed (bits per second)
+        :param can_speed: The speed of the CAN Bus in bits per second
+        
+        """
+        self.can_speed = can_speed 
+    
+    def frame_length(self, frame_data: tuple) -> float:
```

Comment:
This function name suggests that it calculates the length of a single frame, but you actually multiply it by the frequency. Either change the name of the function or multiple by the frequency in the other function (I think the second is the better option)